fix introspection query if any enum column present in primary key (fix #5200) (#5522)
hasura/graphql-engine: CHANGELOG.md

(Add entries here in the order of: server, console, cli, docs, others)

> - server: bugfix for failing introspection if any enum column present in primary key (fixes #5200)

Reviewer: ```suggestion
- server: fix failing introspection query when an enum column is part of a primary key (fixes #5200)
```
